### DISCUSSION TITLE
Issue on github mobile related with profile button & account

### DISCUSSION THREAD

**Discussion by jeftedariel** [community/community#134389] [Mobile] [answered]
Idk if it is the right place to post an issue but when you rename your account username, the account icon on the mobile app won't get updated, and it will still have the old username linked. This causes an error when trying to load the profile. However, if you sign out and then log in again with your account, the icon/button gets updated, and everything works as usual.

This button.
![IMG_20240802_010639.jpg](https://github.com/user-attachments/assets/e7610f1f-5fce-4cbe-bfab-c38379b1b4ba)

**Comment by kangarruu** [comment, marked as answer]
Thanks for writing in @jeftedariel, I could definitely see how that could be confusing! Changing your username can have [unintended side effects](https://docs.github.com/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-user-account-settings/changing-your-github-username). I've updated our team internally to more broadly surface that you need to log out of your GitHub Mobile app to reflect those changes. Thanks for being part of the community!

**Comment by User541f1f1** [comment]
> Idk if it is the right place to post an issue but when you rename your account username, the account icon on the mobile app won't get updated, and it will still have the old username linked. This causes an error when trying to load the profile. However, if you sign out and then log in again with your account, the icon/button gets updated, and everything works as usual.
> 
> 
> This button.
> ![IMG_20240802_010639.jpg](https://github.com/user-attachments/assets/e7610f1f-5fce-4cbe-bfab-c38379b1b4ba)
> 
>
